libs/components/src/lib/switch/switch.ts

```diff
@@ -31,4 +32,78 @@ export class Switch extends FoundationSwitch {
 	 * HTML Attribute: connotation
 	 */
 	@attr connotation?: SwitchConnotation;
+
+	/**
+	 * When true, the control will be immutable by user interaction. See {@link https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/readonly | readonly HTML attribute} for more information.
+	 * @public
+	 * @remarks
+	 * HTML Attribute: readonly
+	 */
+	// @ts-expect-error Type is incorrectly non-optional
+	@attr({ attribute: 'readonly', mode: 'boolean' }) readOnly: boolean; // Map to proxy element
+	// @ts-expect-error Function is declared but never read
+	private readOnlyChanged(): void {
+		if (this.proxy instanceof HTMLInputElement) {
+			this.proxy.readOnly = this.readOnly;
+		}
+
+		this.readOnly
+			? this.classList.add('readonly')
+			: this.classList.remove('readonly');
+	}
+
+	/**
+	 * The element's value to be included in form submission when checked.
+	 * Default to "on" to reach parity with input[type="checkbox"]
+	 *
+	 * @internal
+	 */
+	override initialValue = 'on';
+
+	/**
+	 * @internal
+	 */
+	// @ts-expect-error Type is incorrectly non-optional
+	@observable defaultSlottedNodes: Node[];
```

Comment:
I think for internals we should just do:
```suggestion
	@observable defaultSlottedNodes!: Node[];
```